meta-resin-common/recipes-containers/docker-disk/files/entry.sh

```diff
@@ -11,7 +11,7 @@ PARTITION_SIZE=${PARTITION_SIZE:=1024}
 # Create sparse file to hold ext4 resin-data partition
 dd if=/dev/zero of=/export/data_disk.img bs=1M count=0 seek=$PARTITION_SIZE
 # now partition the newly created file to ext4
-mkfs.ext4 -E lazy_itable_init=0,lazy_journal_init=0 -F /export/data_disk.img
+mkfs.ext4 -E lazy_itable_init=0,lazy_journal_init=0 -i 8192 -F /export/data_disk.img
```

Comment:
Could we possibly use the long form to make it more descriptive. I bet we'll forget what `-i` does in a few weeks